psalm.baseline.xml

```diff
@@ -51,6 +49,10 @@
       <code>string[]</code>
       <code>string[][][]</code>
     </MixedInferredReturnType>
+    <MixedPropertyTypeCoercion occurrences="2">
+      <code>$this-&gt;styleAttributesForNodes</code>
+      <code>$this-&gt;visitedNodes</code>
+    </MixedPropertyTypeCoercion>
```

Comment:
I don't understand why we now have these errors but didn't before.  Though I note the overall error count has gone down :)

Reply:
Now that the field types are more specific, Psalm can recognize cases where the code (or - more probable - the annotations for that code) does not make sure (yet) that these more specific type restrictions are adhered to.

Reply:
OK.